jina/executors/evaluators/__init__.py

```diff
@@ -0,0 +1,21 @@
+__copyright__ = "Copyright (c) 2020 Jina AI Limited. All rights reserved."
+__license__ = "Apache-2.0"
+
+from typing import Sequence
+from jina.proto import jina_pb2
+
+from .. import BaseExecutor
+
+
+class BaseEvaluator(BaseExecutor):
+    """A :class:`BaseEvaluator` is used to evaluate different messages coming from any kind of executor
+    """
+
+    def __init__(self, *args, **kwargs):
+        """"
+        :param id_tag: the key in the tags of documents to identify uniquely a document
+        """
+        super().__init__(*args, **kwargs)
```

Comment:
```suggestion
    def __init__(self, *args, **kwargs):
        """"
        :param id_tag: the key in the tags of documents to identify uniquely a document
        """
        super().__init__(*args, **kwargs)
```

`id_tag` is missing or this part is unnecessary